### PR TITLE
Core/task 228 development environment docker compose

### PR DESCRIPTION
**Pull request by isaac-fibonacci**
1. I added docker-compose.local.yml to manage the containers we need (backend code, postgres, redis).
2. I added nodemon configs to the package.json file
3. I added dependency packages to package.json

**Note**: We share all the code with the container in this docker-compose. After each file change in the src directory, the nodemon package is called, and the recompile container contains the code.

## Checklist

- [ ] Make Sure all containers run correctly and get an IP for each of them
- [ ] Make sure to recompile the code after each file change and your changes will take effect
- [ ] All Merge Request pipelines should be passed.
- [ ] Could you ensure all the necessary changes are correct?
- [ ] Make sure no change is missing (all necessary changes are made).
- [ ] Merge Request should contain only changes for one purpose (one feature, bug, or refactoring).
- [ ] All necessary unit tests are written.
- [ ] Critical changes should be tested locally (or on staging).

### CODE REVIEW

**Review by FlokiBB — APPROVED** [review]

File: ts-version/.sequelizerc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const path = require('path')
-require('dotenv').config('./.env');
```
Comment (FlokiBB) [inline ts-version/.sequelizerc:2]: why our env in sequlize is 
'env': 'development'

for both development and production ?

**Review by FlokiBB — CHANGES_REQUESTED** [review]

**Comment by FlokiBB** [comment]
@isaac-fibonacci did you check rebuild feature on file change?

File: ts-version/.sequelizerc (hunk 1 above)
Comment (isaac-fibonacci) [reply ts-version/.sequelizerc:2]: Done

File: ts-version/localEnv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[1;34m'
+echo -e "${YELLOW}------------------------------------------------------------------------
+------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (isaac-fibonacci) [reply ts-version/localEnv]: Done

**Pull request merged**